pkg/cmd/server/server.go

```diff
@@ -108,6 +111,10 @@ type Config struct {
 	TelemetryInterval        time.Duration
 }
 
+const defaultBackendsPerKey = 1
```

Comment:
it would be nice to make this configurable too while we're here

Reply:
it's not very useful with smaller clusters, but with larger clusters it lets you spread the load for a hot key between multiple nodes